cumulusci/core/tests/test_config.py

```diff
@@ -954,3 +954,51 @@ def test_load_orginfo(self):
         self.assertEqual("Enterprise Edition", config.org_type)
```

Comment:
Looks like you need to declare utf-8 encoding for this file at the top for Python 2